package.json

```diff
@@ -2,12 +2,15 @@
   "private": true,
   "scripts": {
     "bootstrap": "npx lerna bootstrap",
+    "bootstrap_windows": "npx lerna bootstrap --ignore-scripts",
```

Comment:
Did you give thought to a solution here that doesn't require separate `*_windows` scripts?

Reply:
The main reason I added seperate *_windows scripts is that we have to use powershell in windows. Travis will not run bash. There is git bash, but I am not sure if it behaves just liek original bash. I will have a look.  There will be also some small differences in running mocha on windows where we use --recursive flag instead */** notation to run all tests. If git bash will behave same as bash we can change all test scripts in packages to call bash script and there I can have different options for windows and linux. 

I will have a look at it.